packages/components/nodes/textsplitters/LatexTextSplitter/LatexTextSplitter.ts

```diff
@@ -0,0 +1,54 @@
+import { INode, INodeData, INodeParams } from '../../../src/Interface'
```

Comment:
LatexTextSplitter shouldnt be part of this PR, if you can remove these 2 files @Pravesh-mansharamani